### PR TITLE
MOTOR-1031 Changelog for 3.1

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -6,9 +6,24 @@ Changelog
...
+- Support for MongoDB 6.0.
+- Added the following key management APIs to :class:`~motor.core.MotorClientEncryption`:
+  :meth:`~motor.core.MotorClientEncryptio.get_key`
+  :meth:`~motor.core.MotorClientEncryptio.get_keys`
+  :meth:`~motor.core.MotorClientEncryptio.delete_key`
+  :meth:`~motor.core.MotorClientEncryptio.add_key_alt_name`
+  :meth:`~motor.core.MotorClientEncryptio.get_key_by_alt_name`
+  :meth:`~motor.core.MotorClientEncryptio.remove_key_alt_name`
+  :meth:`~motor.core.MotorClientEncryptio.rewrap_many_data_key`
+- Change streams support for user-facing PIT pre- and post-images.
```
Comment (ShaneHarvey) [inline doc/changelog.rst:24]: Let's mention the new argument name in this line.

File: doc/changelog.rst [hunk 2]
```diff
@@ -6,9 +6,24 @@ Changelog
...
+- Allow cursor to be used in async with-statement.
+
 The new Queryable Encryption changes that are in beta are:
 
 - The `encrypted_fields` argument to the
```
Comment (ShaneHarvey) [inline doc/changelog.rst:29]: Can you add the standard `Issues Resolved` section with a link to the jira release notes?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: doc/changelog.rst (hunk 1 above)
Comment (blink1073) [reply doc/changelog.rst]: Done

File: doc/changelog.rst (hunk 2 above)
Comment (blink1073) [reply doc/changelog.rst:30]: Done

**Pull request merged**